### PR TITLE
Automatically tag applications upon submission

### PR DESCRIPTION
**Pull request by dergigi**
Building up on #66. Will help us sort through applications much more easily.

### CODE REVIEW

**Comment by dergigi** [comment]
Manual submission test on deployment preview was successful ✅

File: components/ApplicationForm.tsx [hunk 1]
```diff
@@ -51,6 +67,19 @@ export default function ApplicationForm() {
...
+            <label>
+                Main Focus
+                <small>
+                    In which area will your project have the most impact?
+                </small>
+                <select {...register('main_focus')} >
+                    <option value="bitcoin">Bitcoin</option>
+                    <option value="lightning">Lightning</option>
+                    <option value="nostr">nostr</option>
+                    <option value="other">Other</option>
```
Comment (islandbitcoin) [inline components/ApplicationForm.tsx:79]: should we have an option for legal defense?

Comment (dergigi) [reply components/ApplicationForm.tsx:79]: No.

The "Legal Defense Fund" is its own fund. It is not a bucket of projects that will have its own subcommittee to evaluate project applications.

**Review by h4rpr — APPROVED** [review]
awesome!

**Pull request merged**